opentelemetry-sdk/benches/metric_counter.rs

```diff
@@ -69,6 +70,17 @@ fn counter_add(c: &mut Criterion) {
     });
 }
 
-criterion_group!(benches, criterion_benchmark);
+#[cfg(not(target_os = "windows"))]
```

Comment:
unrelated change, sorry! Hope its minimal (copy from other benches) and hence okay!